pkg/controller/issueapi/issue.go

```diff
@@ -106,12 +106,19 @@ func (c *Controller) IssueMany(ctx context.Context, requests []*IssueRequestInte
 	defer c.recordStats(ctx, results)
 
 	// Send SMS messages if there's an SMS provider.
-	smsProvider, err := c.smsProviderFor(ctx, realm)
+	smsProvider, err := c.smsProviderFor(ctx, realm, false)
```

Comment:
I'm not a huge fan of arbitrary booleans that control behavior. Can we make a helper off of Realm like `smsUserReportProviderFor` instead? The arbitrary true/false is going to cause a bug somewhere.

Reply:
replaced w/ Options pattern. bool was for speedy POC & testing.